vicon_mujoco_init/robot_init.py

```diff
@@ -0,0 +1,155 @@
+import math
+import time
+import context
+import o80
+import pam_mujoco
+import vicon_transformer
+import numpy as np
+from scipy.spatial.transform import Rotation as R
+
+# to run this demo, start pam_mujoco with mujoco_id "vicon_robot_init"
+# pam_mujoco vicon_robot_init && python3 <pathToCode>/pam_demos/vicon_mujoco_init/robot_init.py
+
+# read a test vicon frame from file
+print('read a test vicon frame from file')
+vj = vicon_transformer.ViconJson()
```

Comment:
I assume testViconFrame.txt is read here ?

Reply:
(looks like this demo also requires a "zmq broadcaster" active ... should be mentioned somewhere)